File: Java/Easy/Find-First-Palindromic-String-in-the-Array.java
class Solution {
  public String firstPalindrome(String[] words) {
    for (String word : words) {
      if (checkPalidrome(word)) {
        return word;
      }
    }
    return "";
  }

  public boolean checkPalidrome(String word) {
    for (int i = 0; i <= word.length() / 2; i++) {
      if (word.charAt(i) != word.charAt(word.length() - 1 - i)) {
        return false;
      }
    }
    return true;
  }
}

File: Python/Easy/Find-First-Palindromic-String-in-the-Array.py
class Solution(object):
    def firstPalindrome(self, words):
        """
        :type words: List[str]
        :rtype: str
        """
        for word in words:
            if self.checkPalindrome(word):
                return word
        return ""
    def checkPalindrome(self, word):
        n = len(word)
        for i in range(n):
            if word[i]!=word[n-i-1]:
                return False
        return True
